lib/firespring_dev_commands/aws/route53.rb

```diff
@@ -0,0 +1,107 @@
+module Dev
+  class Aws
+    # Class for performing Route53 functions
+    class Route53
+      attr_reader :client, :zones
+
+      def initialize
+        @client = ::Aws::Route53::Client.new
+        @zones = nil
```

Comment:
Seeing how this is all organized now I feel like we should pass "domains" into the initializer and not passing it to the 'zones' call. Then all of the internal calls in this class should change from `@zones` to just `zones`. We could probably add `private` before the all_zones` and `zones_by_domains_names` methods too, if desired.